Makefile

```diff
@@ -13,12 +13,14 @@ OPEN_URL=xdg-open
 MIGRATOR=$(PENV_BIN_PATH)alembic -c config/alembic.ini
 BWR=bower
 NPM=npm
+MJML=mjml
```

Comment:
@anxolerd the same reason, why there are predefined $(CC) and $(CXX) aliases. For more easy possibility to override them, if they won't be in PATH